pkgs/misc/emulators/sameboy/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, gtk3, rgbds, SDL2, wrapGAppsHook, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "sameboy";
+  version = "0.13.6";
+
+  src = fetchFromGitHub {
+    owner = "LIJI32";
+    repo = "SameBoy";
+    rev = "v" + version;
```

For consistency throughout the codebase:
```suggestion
    rev = "v${version}";
```